schema.py

```diff
@@ -10,5 +10,9 @@ class PredictPneumoniaResponseSchema(Schema):
     outcome = fields.Int()
 
 
+class PredictRespiratoryResponseSchema(Schema):
+    respiratory_disease = fields.String()

```

Comment:
change to `outcome` for consistency throughout, so Jeff got an easier time on the frontend